core/services/ocrcommon/telemetry_test.go

```diff
@@ -598,27 +598,30 @@ func TestCollectMercuryEnhancedTelemetry(t *testing.T) {
 	}
 
 	expectedTelemetry := telem.EnhancedEAMercury{
-		DataSource:                    "data-source-name",
-		DpBenchmarkPrice:              123456.123456,
-		DpBid:                         1234567.1234567,
-		DpAsk:                         321123,
-		CurrentBlockNumber:            123456789,
-		CurrentBlockHash:              common.HexToHash("0x123321").String(),
-		CurrentBlockTimestamp:         987654321,
-		BridgeTaskRunStartedTimestamp: trrsMercury[0].CreatedAt.UnixMilli(),
-		BridgeTaskRunEndedTimestamp:   trrsMercury[0].FinishedAt.Time.UnixMilli(),
-		ProviderRequestedTimestamp:    92233720368547760,
-		ProviderReceivedTimestamp:     -92233720368547760,
-		ProviderDataStreamEstablished: 1,
-		ProviderIndicatedTime:         -123456789,
-		Feed:                          common.HexToHash("0x111").String(),
-		ObservationBenchmarkPrice:     111111,
-		ObservationBid:                222222,
-		ObservationAsk:                333333,
-		ConfigDigest:                  "0200000000000000000000000000000000000000000000000000000000000000",
-		Round:                         22,
-		Epoch:                         11,
-		AssetSymbol:                   "USD/LINK",
+		DataSource:                      "data-source-name",
+		DpBenchmarkPrice:                123456.123456,
+		DpBid:                           1234567.1234567,
+		DpAsk:                           321123,
+		CurrentBlockNumber:              123456789,
+		CurrentBlockHash:                common.HexToHash("0x123321").String(),
+		CurrentBlockTimestamp:           987654321,
+		BridgeTaskRunStartedTimestamp:   trrsMercury[0].CreatedAt.UnixMilli(),
+		BridgeTaskRunEndedTimestamp:     trrsMercury[0].FinishedAt.Time.UnixMilli(),
+		ProviderRequestedTimestamp:      92233720368547760,
+		ProviderReceivedTimestamp:       -92233720368547760,
+		ProviderDataStreamEstablished:   1,
+		ProviderIndicatedTime:           -123456789,
+		Feed:                            common.HexToHash("0x111").String(),
+		ObservationBenchmarkPrice:       111111,
+		ObservationBid:                  222222,
+		ObservationAsk:                  333333,
+		ConfigDigest:                    "0200000000000000000000000000000000000000000000000000000000000000",
+		Round:                           22,
+		Epoch:                           11,
+		AssetSymbol:                     "USD/LINK",
+		ObservationBenchmarkPriceString: "111111",
+		ObservationBidString:            "222222",
+		ObservationAskString:            "333333",
```

Comment:
@george-dorin What is the plan (assuming there is one) for dropping support for the old fields from the proto?

Reply:
Thinking about this a bit more 🤔 If the current value is garbage, is there any harm in dropping the old field?

Old code would still be sending the old field, so as long as the telemetry pipeline can interpret both we shouldn't have a problem.

Reply:
There is a mechanism for reserving the deprecated field: https://protobuf.dev/programming-guides/proto3/#deleting

Reply:
The plan is to reserve them further down the line so we don't have to have two versions of the protobuf in the ingestion server.